src/simulations/runner.py

```diff
@@ -553,7 +552,11 @@ def run(run_mode, settings, no_runs=None):
             # with relevant parameters
             record.init_env(env)  # record progress as we train
             runner = Runner(env, prm, record)
-            runner.run_experiment()
+            runner.run_experiment(prm)
+            record.timer_stats(prm, runner.explorer.env.network.timer_pp,
+                               runner.explorer.env.network.timer_comparison,
+                               runner.explorer.data.timer_optimisation,
+                               runner.explorer.data.timer_feasible_data, i)
```

Comment:
you do not need to add `i` as an input. These will be saved in the `outputs/results/run{no_run}/record` folder. It will make it harder to access the files if the file names have random index extensions to them.

Reply:
I've now corrected this function so that it makes more sense. It took care of the `pop(0)`, removed the number of run as parameter and polished the string :)